sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -1987,3 +1988,84 @@ case class GpuStringInstr(str: Expression, substr: Expression)
     }
   }
 }
+
+
+
+class GpuConvMeta(
+  expr: Conv,
+  conf: RapidsConf,
+  parent: Option[RapidsMeta[_,_,_]],
+  rule: DataFromReplacementRule) extends TernaryExprMeta(expr, conf, parent, rule) {
+
+  override def tagExprForGpu(): Unit = (expr.fromBaseExpr, expr.toBaseExpr) match {
+    case (Literal(fromBaseVal, IntegerType), Literal(toBaseVal, IntegerType))
+      if Set(fromBaseVal, toBaseVal).subsetOf(Set(10, 16)) => ()
+    case _ =>
+      willNotWorkOnGpu("only literal 10 or 16 for from_base and to_base are supported")
+  }
+
+  override def convertToGpu(
+    numStr: Expression,
+    fromBase: Expression,
+    toBase: Expression): GpuExpression = GpuConv(numStr, fromBase, toBase)
+}
+
+
+case class GpuConv(num: Expression, fromBase: Expression, toBase: Expression)
+  extends GpuTernaryExpression {
+
+  override def doColumnar(
+    v1: GpuColumnVector,
+    v2: GpuColumnVector,
+    v3: GpuColumnVector): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(v1: GpuScalar, v2: GpuColumnVector, v3: GpuColumnVector): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(v1: GpuScalar, v2: GpuScalar, v3: GpuColumnVector): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(v1: GpuScalar, v2: GpuColumnVector, v3: GpuScalar): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(v1: GpuColumnVector, v2: GpuScalar, v3: GpuColumnVector): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(v1: GpuColumnVector, v2: GpuColumnVector, v3: GpuScalar): ColumnVector = {
+    throw new UnsupportedOperationException()
+  }
+
+  override def doColumnar(
+    numRows: Int,
+    str: GpuScalar,
+    fromBase: GpuScalar,
+    toBase: GpuScalar): ColumnVector = {
+      throw new UnsupportedOperationException()
```

Comment:
We have to support this. There are corner cases with ScalarSubquery that this would blow up if we don't support it.